.github/workflows/makefile.yml

```diff
@@ -35,14 +35,34 @@ jobs:
         sudo apt-get update
         sudo apt-get install docker-ce docker-ce-cli containerd.io
 
-    - name: IPDK Ubuntu Container Build
+    - name: IPDK Ubuntu 20.04 Container Build
       run: |
         cd build
-        ./ipdk install
-        cat << EOF >> ~/.ipdk/ipdk.env
-        # When using ubuntu20.04
-        BASE_IMG=ubuntu:20.04
-        IMAGE_NAME=ipdk/p4-ovs-ubuntu20.04
-        DOCKERFILE=${GITHUB_WORKSPACE}/build/Dockerfile.ubuntu
-        EOF
-        ipdk build --no-cache
+        ./ipdk install ubuntu2004
+        ipdk build --no-cache 
+
+  build_Fedora33:
+    needs: shellcheck
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        sudo echo 'debconf debconf/frontend select Noninteractive' | sudo debconf-set-selections
+        sudo apt-get -y install apt-utils
+
+    - name: Install Docker CE
+      run: |
+        sudo apt-get install ca-certificates curl gnupg lsb-release
+        curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+        echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+        sudo apt-get update
+        sudo apt-get install docker-ce docker-ce-cli containerd.io
+
+    - name: IPDK Fedora 33 Container Build
+      run: |
+        cd build
+        ./ipdk install fedora33
```

Comment:
@stolsma The ubuntu/fedora github actions look very similar. Could we use a [matrix](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-running-with-multiple-operating-systems)?
For example:

```yaml
strategy:
  matrix:
    os: [ubuntu2004, fedora33]

...
./ipdk install ${{ matrix.os }}
```

Reply:
I like this a a lot, thanks for proposing this @rst0git! @stolsma can you make this change?

Reply:
Yeah! Great comment! Did know about that option but just forgot about it! Thanks @rst0git ! 👍